schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml

```diff
@@ -0,0 +1,57 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x kvm
```

Comment:
s/kvm/kvm textmode/

Reply:
Don't create new yaml file, reuse the default_s390x_kvm_textmode.yaml.